app/gui/src/controller/visualization/manager.rs

```diff
@@ -158,6 +158,7 @@ impl Default for Status {
 #[allow(missing_docs)]
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct Desired {
+    pub module:           String,
```

Comment:
Should be `QualifiedName`.

Reply:
Done in [8f60565](https://github.com/enso-org/enso/pull/7115/commits/8f60565a0b981a02ef433cbf1db9b5fb8e116225)